assets/tests/components/detours/detourListPage.test.tsx

```diff
@@ -1,14 +1,14 @@
 import { describe, test, expect, jest, beforeEach } from "@jest/globals"
 import React from "react"
-import { DetourListPage } from "../../src/components/detourListPage"
-import { fetchDetours } from "../../src/api"
-import { neverPromise } from "../testHelpers/mockHelpers"
-import { Ok } from "../../src/util/result"
+import { DetourListPage } from "../../../src/components/detourListPage"
+import { fetchDetours } from "../../../src/api"
+import { neverPromise } from "../../testHelpers/mockHelpers"
+import { Ok } from "../../../src/util/result"
 import { render, screen } from "@testing-library/react"
 
-jest.useFakeTimers().setSystemTime(new Date("2024-08-29T20:00:00"))
+jest.spyOn(Date, "now").mockImplementation(() => 1724961600000)
```

Comment:
question: what's the benefit of this change? Personally, I liked the previous way and felt it was a better way of doing things, and I'd like to update the rest of Skate to use `useFakeTimers`, was there something that necessitated this change? :o

Reply:
Oh!! Ok, so the `useFakeTimers` wasn't working for the new tests I wrote, and then I was reverse-influenced by how we use the spyOn approach everywhere else. I'll revert this change at least, and then see if I can get `useFakeTimers` to work in the new `openDetour` tests

Reply:
Fixed!